<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>775</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>200</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="controlWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>315</width>
         <height>400</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QScrollArea" name="scrollArea_3">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>493</width>
             <height>700</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>3</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QTabWidget" name="Control_tab">
              <property name="tabPosition">
               <enum>QTabWidget::West</enum>
              </property>
              <property name="tabShape">
               <enum>QTabWidget::Triangular</enum>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="layer">
               <attribute name="title">
                <string>Layer</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_5">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <property name="horizontalSpacing">
                 <number>-1</number>
                </property>
                <item row="2" column="0">
                 <widget class="QGroupBox" name="groupBox_image_control">
                  <property name="title">
                   <string>Image Control</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>2</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>2</number>
                   </property>
                   <item>
                    <widget class="QComboBox" name="control_image_type_3">
                     <item>
                      <property name="text">
                       <string>No difference images</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Running difference</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Running difference Ratio</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Base difference</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Base difference Ratio</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QGroupBox" name="groupBox_image_layers">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="title">
                   <string>Image Layer</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>4</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>4</number>
                   </property>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout">
                     <item>
                      <widget class="QPushButton" name="NewLayer">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="minimumSize">
                        <size>
                         <width>127</width>
                         <height>32</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>New Layer</string>
                       </property>
                       <property name="icon">
                        <iconset>
                         <normaloff>icon/plus-symbol-button.svg</normaloff>icon/plus-symbol-button.svg</iconset>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="horizontalSpacer_2">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeType">
                        <enum>QSizePolicy::MinimumExpanding</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>150</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <widget class="QScrollArea" name="scrollArea">
                     <property name="widgetResizable">
                      <bool>true</bool>
                     </property>
                     <widget class="QWidget" name="LayerlistBox">
                      <property name="geometry">
                       <rect>
                        <x>0</x>
                        <y>0</y>
                        <width>453</width>
                        <height>483</height>
                       </rect>
                      </property>
                     </widget>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QGroupBox" name="groupBox_movie_controls">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="title">
                   <string>Movie controls</string>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_10">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>2</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>2</number>
                   </property>
                   <item row="2" column="0">
                    <widget class="QPushButton" name="btn_prev">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>30</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>20</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="statusTip">
                      <string>Step to previous frame</string>
                     </property>
                     <property name="icon">
                      <iconset>
                       <normaloff>icon/left-arrow-4.svg</normaloff>icon/left-arrow-4.svg</iconset>
                     </property>
                     <property name="shortcut">
                      <string>Left</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="0" colspan="5">
                    <widget class="QSlider" name="Slider_frame">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="4">
                    <widget class="QLabel" name="label_frame">
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="2">
                    <widget class="QPushButton" name="btn_next">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>30</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>30</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="statusTip">
                      <string>Step to next frame</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset>
                       <normaloff>icon/right-arrow-2.svg</normaloff>icon/right-arrow-2.svg</iconset>
                     </property>
                     <property name="shortcut">
                      <string>Right</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="1">
                    <widget class="QPushButton" name="btn_play">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>30</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>30</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="statusTip">
                      <string>Play movie</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset>
                       <normaloff>icon/right-arrow.svg</normaloff>icon/right-arrow.svg</iconset>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="3">
                    <spacer name="horizontalSpacer">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeType">
                      <enum>QSizePolicy::Expanding</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>120</width>
                       <height>12</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="plotWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>240</width>
         <height>0</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1023</width>
     <height>22</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
    <addaction name="actionDisplay_Header"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>solarpy</string>
    </property>
    <addaction name="actionsolarpy"/>
   </widget>
   <widget class="QMenu" name="menuModules">
    <property name="title">
     <string>Modules</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuModules"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExit"/>
   <addaction name="actionOpen"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actiontools"/>
   <addaction name="separator"/>
   <addaction name="actionLoadChunk"/>
   <addaction name="actionslit"/>
   <addaction name="actionTrack"/>
   <addaction name="actiontestaction"/>
  </widget>
  <action name="actionOpen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icon/open-folder-with-document.svg</normaloff>icon/open-folder-with-document.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="statusTip">
    <string>Open new File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>icon/power-symbol-2.svg</normaloff>icon/power-symbol-2.svg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Exit application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionsolarpy">
   <property name="text">
    <string>About solarpy</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normaloff>icon/undo-arrow2.svg</normaloff>icon/undo-arrow2.svg</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="statusTip">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset>
     <normaloff>icon/redo-arrow-2.svg</normaloff>icon/redo-arrow-2.svg</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="statusTip">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionslit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icon/map-route.svg</normaloff>icon/map-route.svg</iconset>
   </property>
   <property name="text">
    <string>Add slit</string>
   </property>
   <property name="toolTip">
    <string>Add slit</string>
   </property>
   <property name="statusTip">
    <string>Add a slit</string>
   </property>
  </action>
  <action name="actionTrack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icon/aim.svg</normaloff>icon/aim.svg</iconset>
   </property>
   <property name="text">
    <string>Track</string>
   </property>
   <property name="toolTip">
    <string>Track</string>
   </property>
   <property name="statusTip">
    <string>De-rotation</string>
   </property>
  </action>
  <action name="actionLoadChunk">
   <property name="icon">
    <iconset>
     <normaloff>icon/download.svg</normaloff>icon/download.svg</iconset>
   </property>
   <property name="text">
    <string>LoadChunk</string>
   </property>
   <property name="statusTip">
    <string>LoadChunk</string>
   </property>
  </action>
  <action name="actionDisplay_Header">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Display Header</string>
   </property>
  </action>
  <action name="actiontestaction">
   <property name="text">
    <string>testaction</string>
   </property>
  </action>
  <action name="actiontools">
   <property name="icon">
    <iconset>
     <normaloff>icon/edit.svg</normaloff>icon/edit.svg</iconset>
   </property>
   <property name="text">
    <string>tools</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
